src/client/styles/themes/_grid-layouts.sass

```diff
@@ -0,0 +1,54 @@
+// overlay
+body.layout-overlay
+	#videos-grid 
+		article.video
+			padding: 0
+
+			button.cue
+				z-index: 1
+				right: 4px
+
+				&:hover~header
```

Comment:
This is clearly not the neatest way of solving the issue. We'll fix that when we change the button design